armory/datasets/download.py

```diff
@@ -0,0 +1,76 @@
+from pathlib import Path
+
+import boto3
+from botocore import UNSIGNED
+from botocore.client import Config
+from botocore.exceptions import ClientError
+
```

Comment:
These should be added in `pyproject.toml` under `datasets`.

Reply:
Both `boto3` and `botocore`.

Reply:
Should we have them in both datasets and engine?

Reply:
No issue there. Let `pip` take the wheel.

Reply:
Done.